lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -133,7 +128,7 @@ def main():
                 os.remove(output_file)
             else:
                 print(f"\n--> Stop")
-                exit(1)
+                exit(0)
```

Comment:
NO! don't exit(0) on error conditions.

Reply:
it is not an error

Reply:
it is just replying "no" to a question

Reply:
Like I said, the script did not finish it's task successfully, because of user intervention. That should not have an exit code of 0.